content/en/blog/2023/gc-candidates.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Final list of candidates for the 2023 OpenTelemetry Governance Committee
+linkTitle: 2023 GC Candidates
+date: 2023-10-19
+author: OpenTelemetry Governance Committee
+aliases: [/blog/2023/gc-elections-2023-list-of-candidates]
+cSpell:ignore: Blanco Sirianni Vijay
+---
+
+The OpenTelemetry election committee is pleased to announce the final list of
+candidates running for one of the four available seats. We encourage all voters
+to take a moment and review all candidates, picking the one that best represents
+your interest. You can find their pictures, profile link, and descriptions on
+the
+[candidates page](https://github.com/open-telemetry/community/blob/main/elections/2023/governance-committee-candidates.md),
+but here are their names:
+
+- Austin Parker
+- Daniel Gomez Blanco
+- Johannes Tax
+- Juraci Paixão Kröhling
+- Eric Sirianni
+- Vijay Samuel
+- Severin Neumann
+- Ted Young
+- Steve Flanders
+
+You can check your eligibility by reviewing
+[this GitHub issue](https://github.com/open-telemetry/community/issues/1561). If
+you are not listed there but believe you have the right to vote, please
+[fill out this registration form](https://forms.gle/GWuGZKku326pCLUo6).
```

Comment:
The form seems to be closed. Should it be open already?

Reply:
Fixed